content/2023/chinese/_index.md

```diff
@@ -1,35 +1,66 @@
+
+At this Jakarta One Livestream conference, you will have the opportunity to discuss the development trends of the cloud native era with top global technical experts and developers, and jointly move towards key milestones in the cloud native era< Br>
+
+Whether you are an experienced Java developer or an emerging cloud native application developer, JakartaOne Livestream will be a technology feast that you cannot miss. Register now and join us to explore the infinite possibilities of the cloud native era together< Br>
+
+The Chinese version of this meeting is provided by [Jakarta EE China Community]（ https://jakarta.ee/zh/community/china/ ）Sponsored by the Eclipse Foundation and initiated by Primeton Information, the Jakarta EE China Community is jointly built and developed by enterprises such as AsiaInfo, IBM, Tongtech, Kingdee, and Cvicse, as well as individual developers. It is committed to promoting Jakarta EE's awareness and technology sharing in the Greater China region< Br>
+
+[Register]（ https://meeting.tencent.com/dw/OYvhdc8AQsTK?ch=nxgCi8mdbizBv ）You can participate in the lucky draw at the event site for this seminar.		{{</ events/registration >}}
     {{</ grid/div >}}
 {{</ grid/section-container >}}
 
+{{< grid/section-container id="sponsors" class="backdrop">}}
+  {{< events/sponsors event="chinese" year="2023" title="Online Lucky Draw" displayBecomeSponsor="false" >}}
+{{</ grid/section-container >}}
+
+<!-- Add agenda using legacy CSS -->
+{{< grid/section-container class="alt" id="plan-of-the-day" containerClass="backdrop padding-top-40" >}}
+  {{< grid/div class="container" isMarkdown="false">}}
+    {{< events/agenda event="chinese" year="2023" >}}
+  {{</ grid/div >}}
+{{</ grid/section-container >}}
+
+<!-- Add speakers section -->
+{{< grid/section-container id="speakers" class="speaker-container text-center">}}
+  {{< grid/div class="padding-top-40" isMarkdown="false">}}
+    {{< events/user_display year="2023" event="chinese" source="speakers" subpage="speakers" useCarousel="false" imageRoot="/images/speakers/" title="演讲人" headerClass="text-left" />}}
+  {{</ grid/div >}}
+{{</ grid/section-container >}}
+
 <!-- Add user carousel for committee -->
 {{< events/program-committee event="chinese" year="2023"  source="committee" useCarousel="false">}}
-We are proud to say that the sessions and keynotes for JakartaOne Livestream Event are chosen by an independent program committee made up of volunteers from the Jakarta&reg; EE and Cloud Native Java community, especially from the Jakarta EE Community China!
-{{</ events/program-committee >}}
+The speech topics and sessions of the Jakarta One Livestream conference were determined by an independent preparatory committee, with all committee members from the Jakarta EE and cloud native Java communities, especially the Jakarta EE China community.{{</ events/program-committee >}}
+<!-- Add modal for use w/ agenda -->
```

Comment:
```suggestion
<!-- Add modal for use w/ agenda -->
{{< bootstrap/modal id="eclipsefdn-modal-event-session" >}}
```

This shortcode is required to display the abstracts in the agenda.

Reply:
Just fixed this in [06d1bf0](https://github.com/jakartaee/jakartaone.org/pull/499/commits/06d1bf0519ad8685e904171f7357a48e89336c53)

Reply:
I think you missed the `{{< bootstrap/modal id="eclipsefdn-modal-event-session" >}}` line. No worries. For the sake of time I'll merge and fix ASAP in another PR.